src/gluonts/time_feature/lag.py

```diff
@@ -83,16 +85,22 @@ def _make_lags_for_month(multiple, num_cycles=3):
     # multiple, granularity = get_granularity(freq_str)
     offset = to_offset(freq_str)
 
-    if offset.name == "A-DEC":  # normalized string for YearEnd aka Y
+    if (
```

Comment:
Can you put these comments on the line before? Black does some weird formatting otherwise :|